### DISCUSSION TITLE
can reth be compiled against the target x86_64-unknown-linux-musl

### DISCUSSION THREAD

**Discussion by MoeMahhouk** [paradigmxyz/reth#7445] [Help]
Hey folks,
I would like to compile the reth binary against the target x86_64-unknown-linux-musl because I would like to run it on a minimal VM.
However, it seems that the database mbdx is having some dependencies that are not compatible with musl-gcc (error: "limits.h file not found" during compilation).
I tried to adjust the compiler in the build.rs in the libmdbx-rs directory but no luck either.
I would appreciate your help if possible.

edit:
I made sure that musl, musl-dev and musl-tools are installed and up-to-data.
Also set the CC env variable to musl-gcc instead.

Alternatively, would it be possible to have a static reth binary?

**Comment by emhane** [comment]
did you check out https://reth.rs/installation/binaries.html
